tests/testthat/test-all-equal.r

```diff
@@ -1,3 +1,7 @@
+test_that("all_equal is deprecated", {
+  expect_snapshot(all_equal(mtcars, mtcars))
```

Comment:
```suggestion
  local_options(lifecycle_verbosity = "warning")
  expect_snapshot(all_equal(mtcars, mtcars))
```

Just in case, for whatever reason, `all_equal()` gets called somewhere else in the global session

Reply:
`local_reproducible_output()` already does that automatically.

Reply:
Neat! Didn't know that